tests/bsim/bluetooth/mesh/src/test_blob.c

```diff
@@ -149,7 +149,7 @@ static void blob_cli_caps(struct bt_mesh_blob_cli *b, const struct bt_mesh_blob_
 		ASSERT_EQUAL(caps->max_size, CONFIG_BT_MESH_BLOB_SIZE_MAX);
 		ASSERT_EQUAL(caps->min_block_size_log, BLOB_BLOCK_SIZE_LOG_MIN);
 		ASSERT_EQUAL(caps->max_block_size_log, BLOB_BLOCK_SIZE_LOG_MAX);
-		ASSERT_EQUAL(caps->max_chunk_size, BLOB_CHUNK_SIZE_MAX(BT_MESH_RX_SDU_MAX));
+		ASSERT_EQUAL(caps->max_chunk_size, MIN(CONFIG_BT_MESH_RX_BLOB_CHUNK_SIZE, CONFIG_BT_MESH_TX_BLOB_CHUNK_SIZE));
```

Comment:
Not needed, but you could put this as a macro at the start of the test since it is used multiple places!

Reply:
I just added it as a static variable up top. Can still change it. Technically these changes aren't needed for current state of testing, just if you were to change the max chunk sizes asymmetrically, this would be at least needed. (some low values need a lot more config changes in the test(s) as then there'll be timeouts etc.